Source/WebCore/svg/SVGAngle.h

```diff
@@ -56,6 +56,11 @@ class SVGAngle : public SVGValueProperty<SVGAngleValue> {
 
     SVGAngleValue::Type unitType() const
     {
+        // Per spec https://svgwg.org/svg2-draft/types.html#__svg__SVGAngle__SVG_ANGLETYPE_UNKNOWN
+        // > If other types of angles are supported and used, the SVGAngle uses
+        // > the SVG_ANGLETYPE_UNKNOWN unit type.
```

Comment:
Typically we don't include long comments in the code. A URL is okay though.

Reply:
fixed